sdk/docs/pages/typescript/utils.mdx

```diff
@@ -0,0 +1,46 @@
+# The `@msyten/sui.js/utils` package
+
+This package contains some utilities that simplify common operations when working with the Sui
+Typescript SDK.
```

Comment:
Not to be that person LOL, but I think this should be `TypeScript`

Reply:
I'll update in the next batch of docs changes